biopet.wdl

```diff
@@ -244,6 +244,39 @@ task SampleConfig {
     }
 }
 
+task SampleConfigCromwellArrays {
+    input {
+        File? toolJar
+        String? preCommand
+        Array[File]+ inputFiles
+        String outputPath
+
+        Int memory = 4
+        Float memoryMultiplier = 2.0
+    }
+
+    String toolCommand = if defined(toolJar)
+        then "java -Xmx" + memory + "G -jar " +toolJar
+        else "biopet-sampleconfig -Xmx" + memory + "G"
+
+    command {
+        set -e -o pipefail
+        ~{preCommand}
+        mkdir -p . ~{"$(dirname " + outputPath + ")"}
```

Comment:
outputPath is required, so why is the `.` in this line? You should always get an argument anyway. Most of what's in the placeholder can be outside of it as well:
`mkdir -p $(dirname ~{outputPath})`